sycl/test/extensions/properties/properties_latency_control.cpp

```diff
@@ -0,0 +1,32 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -fsyntax-only -Xclang -verify -Xclang -verify-ignore-unexpected=note,warning %s
+// expected-no-diagnostics
+
+#include <CL/sycl.hpp>
+
+#include <sycl/ext/oneapi/latency_control/properties.hpp>
+
+using namespace sycl::ext::oneapi::experimental;
+
+int main() {
+  // Check that is_property_key is correctly specialized.
+  static_assert(is_property_key<latency_anchor_id_key>::value);
+  static_assert(is_property_key<latency_constraint_key>::value);
+
+  // Check that is_property_value is correctly specialized.
+  static_assert(is_property_value<decltype(latency_anchor_id<-1>)>::value);
```

Comment:
There is no API like
```suggestion
  static_assert(is_property_value_v<decltype(latency_anchor_id<-1>)>);
```
?
Idem for the key above?

Reply:
Hmm I don't think there is...